README.md

```diff
@@ -51,6 +51,79 @@ Be patient when OpenOCD is flashing. Typically, it takes about 55 seconds.
 Use `make help` to get detailed build targets.
 
 
+# 64-bit System Support
+
+## 1. GNU Toolchain
```

Comment:
Move the parts about GNU Toolchain, compiler, and st-link to section `Prerequisites`.